tests/execute/duration/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    for method in shell.tmt shell.detach; do
+        rlPhaseStartTest "Test $method"
+            rlRun "tmt run -vfi $tmp -a execute -h $method test --name good" 0
+            rlRun "tmt run -vfi $tmp -a execute -h $method test --name long" 2
```

Comment:
`2` ... is that what we expect on `timeout` = errors? I thought of this more like `test failed` screnario. But the view may vary :).

Reply:
This sounds like a clear error to me: There was something wrong, the test did not finish, we cannot rely on the results and have to investigate. @thrix, @lukaszachy, would you expect anything else?

Reply:
Well, you're right. Unless it's a `performance` test suite, test should probably always finish (and not timeout). Not sure how I got that impression before (maybe the heat:)).

Reply:
Good :)